### PR TITLE
Fix Windows 10 installation Appx commands error when pwsh.exe installed

### PR DESCRIPTION
**Pull request by 1552980358**
Issue PowerShell/PowerShell#19031 still exists on Windows 10 and not be fixed.

image

### CODE REVIEW

File: MagiskOnWSA/installer/Install.ps1 [hunk 1]
```diff
@@ -53,10 +57,9 @@ Function Finish {
     Start-Process "wsa://com.android.vending"
 }
 
-If (Test-CommandExist pwsh.exe) {
+If (Check-Windows11 -And Test-CommandExist pwsh.exe) {
```
Comment (MustardChef) [inline MagiskOnWSA/installer/Install.ps1:60]: Use
``` powershell
If ((Check-Windows11) -And (Test-CommandExist 'pwsh.exe')) {
```

**Review by MustardChef — CHANGES_REQUESTED** [review]
Looks good to me. Just need to tweak a small thing.

Thank you, 
Mustard Chef.

File: MagiskOnWSA/installer/Install.ps1 (hunk 1 above)
Comment (1552980358) [reply MagiskOnWSA/installer/Install.ps1]: I have tested, and my PR one works, but using brackets has better readability, so I have updated.

Comment (MustardChef) [reply MagiskOnWSA/installer/Install.ps1]: Thank you.

**Comment by MustardChef** [comment]
LGTM!

**Review by MustardChef — APPROVED** [review]

**Comment by chcoust** [comment]
Hello @MustardChef, 
I faced to issue in latest Win10 version WSABuilds LTS Build #4 (15/07/24 - 2407.40000.0.0) for Windows 10 x64
I think the fix was not merged/pushed to latest build
I applied your Install.ps1 fix to the downloded build and all is fine for me

**Comment by MustardChef** [comment]
> Hello @MustardChef, I faced the issue in latest Win10 version _WSABuilds LTS Build #4 (15/07/24 - 2407.40000.0.0) for Windows 10 x64_ I think the fix was not merged/pushed to latest build I applied your Install.ps1 fix to the downloded build and all is fine for me

Thank you for reporting. Since we switched to the upstream repo for most of the builds, the fix has not been applied to the build build using the scripts from the repo. I will make sure that the fix has been applied in the next LTS builds.

**Pull request merged**